src/Plugin_Command.php

```diff
@@ -760,29 +766,62 @@ protected function get_item_list() {
 			$update_info     = ( isset( $all_update_info->response[ $file ] ) && null !== $all_update_info->response[ $file ] ) ? (array) $all_update_info->response[ $file ] : null;
 			$name            = Utils\get_plugin_name( $file );
 			$wporg_info      = $this->get_wporg_data( $name );
+			$plugin_data     = get_plugin_data( WP_PLUGIN_DIR . '/' . $file, false, false );
 
 			if ( ! isset( $duplicate_names[ $name ] ) ) {
 				$duplicate_names[ $name ] = array();
 			}
 
+			$php_version = PHP_VERSION;
+
+			$requires     = isset( $update_info ) && isset( $update_info['requires'] ) ? $update_info['requires'] : null;
+			$requires_php = isset( $update_info ) && isset( $update_info['requires_php'] ) ? $update_info['requires_php'] : null;
+
+			// If an update has requires_php set, check to see if the local version of PHP meets that requirement
+			// The plugins update API already filters out plugins that don't meet WordPress requirements, but does not
+			// filter out plugins based on PHP requirements -- so we must do that here
+			$compatible_php = empty( $requires_php ) || version_compare( PHP_VERSION, $requires_php, '>=' );
+
+			if ( ! $compatible_php ) {
+				$update                    = 'unavailable';
+				$update_unavailable_reason = "Requires a newer version of PHP [$requires_php] than available [$php_version]";
```

Comment:
Nit/curious: why square brackets instead of parentheses? Is that a common style elsewhere in the project?

Reply:
This is just a personal practice hold over from writing lots of code that goes into log files where square brackets are often used over parentheses. It probably doesn't need either of them here though, will change

Reply:
Does this seem OK:
```
Warning: wp-super-cache: This update requires WordPress version 6.5, but the version installed is 6.2.
```
(Changing the text means changing all the test, so we can figure out a good string here first

Reply:
Looks good to me!